packages/trader/src/AppV2/Components/BottomNav/bottom-nav.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Localize } from '@deriv/translations';
+import {
+    LegacyMarketBasketIndicesIcon,
+    LegacyMenuHamburger1pxIcon,
+    StandaloneChartCandlestickRegularIcon,
+    StandaloneClockThreeRegularIcon,
+} from '@deriv/quill-icons';
+import BottomNavItem from './bottom-nav-item';
+import { Badge } from '@deriv-com/quill-ui';
+
+type BottomNavProps = {
+    className?: string;
+    children: React.ReactNode[];
+};
+
+const BottomNavItems = [
```

Comment:
```suggestion
const bottomNavItem = [
```

Reply:
there are 4 objects in the array thats why its plural

Reply:
I meant should it start with lowercase?

Reply:
oh lol